R/generate_rules.R

```diff
@@ -36,20 +37,20 @@ generate_rules <- function(X, ite_std, ntrees_rf, ntrees_gbm, node_size,
 
   # Random Forest
   set.seed(seed_1)
-  forest <- suppressWarnings(randomForest::randomForest(x = X, y = ite_std,
+  forest <- randomForest::randomForest(x = X, y = ite_std,
                                                         sampsize = sf * N,
-                                                        replace = FALSE,
+                                                        replace = replace,
                                                         ntree = 1,
                                                         maxnodes = mn,
-                                                        nodesize = node_size))
+                                                        nodesize = node_size)
   for(i in 2:ntrees_rf) {
     mn <- 2 + floor(stats::rexp(1, 1 / (max_nodes - 2)))
     set.seed(seed_2[i])
     model1_RF <- suppressWarnings(
```

Comment:
Let's keep consistency and remove the suppressWarnings for all randomforest function calls.